common-lib/src/test/java/com/google/cloud/tools/intellij/BasePluginInfoServiceTest.java

```diff
@@ -31,10 +31,10 @@
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.mockito.Mock;
-import org.mockito.runners.MockitoJUnitRunner;
+import org.mockito.junit.MockitoJUnitRunner;
 
 /** Tests for {@link BasePluginInfoService}. */
-@RunWith(MockitoJUnitRunner.class)
+@RunWith(MockitoJUnitRunner.Silent.class)
```

Comment:
In Mockito 2, the MockitoJUnitRunner enforces strict mocks; if you stub something on a mock that isn't called, it throws an error. The `Silent` runner will not fail the test in this scenario. I figured it might be easier to mark this common base class with `Silent` for now until we begin replacing usages of it with `CloudToolsRule`.